lib/iris/coords.py

```diff
@@ -1639,37 +1710,27 @@ def _new_bounds_requirements(self, bounds):
                 raise ValueError('The direction of monotonicity must be '
                                  'consistent across all bounds')
 
-    @property
-    def bounds(self):
-        """
-        The bounds values as a read-only NumPy array, or None if no
-        bounds have been set.
-
-        """
-        bounds = None
-        if self.has_bounds():
-            bounds = self._bounds_dm.data
-        return bounds
-
-    @bounds.setter
-    def bounds(self, bounds):
-        if bounds is None:
-            self._bounds_dm = None
-        else:
+    def _bounds_setter(self, bounds):
+        if bounds is not None:
             # Ensure we have a realised array of new bounds values.
             copy = is_lazy_data(bounds)
             bounds = as_concrete_data(bounds)
             bounds = np.array(bounds, copy=copy, ndmin=2)
```

Comment:
You no longer need to set `ndmin` here (it'll be done by `_sanitise_points` in the parent).